logisland-core/logisland-api/src/main/java/com/hurence/logisland/validator/StandardValidators.java

```diff
@@ -154,6 +154,12 @@ public ValidationResult validate(final String subject, final String value) {
             return new ValidationResult.Builder().subject(subject).input(value).valid(value != null && !value.isEmpty()).explanation(subject + " cannot be empty").build();
         }
     };
+    public static final Validator BYTES_VALIDATOR = new Validator() {
+        @Override
+        public ValidationResult validate(final String subject, final String value) {
+            return new ValidationResult.Builder().subject(subject).input(value).valid(value.getBytes().length == 16).explanation(subject + " iv must be byte array of 16 bytes").build();
```

Comment:
I thought I will use a validator to test the size of the byte array but since we have different sizes of bytes array depending on the algorithm, I replace the testing to the Classes of the algorithms so I can delete this validator now

Reply:
Ok then just remove this code